R/ancova.R

```diff
@@ -2,14 +2,15 @@
 #' Analysis of Covariance
 #'
 #' Performs an analysis of covariance between two groups returning the estimated
-#' "treatment effect" and least squared means estimates of each group.
+#' "treatment effect" (i.e. the contrast between the two treatment groups) and 
```

Comment:
I think maybe we should remove the "the"  i.e.   "the contrast between two treatment groups"